src/components/DocumentationTopic/PrimaryContent/DeclarationSource.vue

```diff
@@ -203,6 +204,10 @@ export default {
     handleWindowResize() {
       this.displaysMultipleLines = displaysMultipleLines(this.$refs.declarationGroup);
     },
+    extraClassesFor(token) {
+      const { highlightDiff: highlighted = false } = token;
```

Comment:
I'm not familiar with this syntax. Does this mean that it will load the `highlightDiff` field from `token` into `highlighted`, setting it to `false` if it's undefined?

Reply:
Yes, this is a [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment), taking the `highlightDiff` key and assigning it to a new const called `highlighted` with a default value of `false`, in case `highlightDiff` is undefined

Reply:
Awesome, thanks for the info and the link!